airgun/widgets.py

```diff
@@ -707,6 +767,81 @@ def fill(self, values):
             self.add(param)
 
 
+class InheritedGlobalParameter(Table):
```

Comment:
why do we need separate class for that table? and why do we put it in common widgets?

Yes, we can think about creating widget for new control, like:
![image](https://user-images.githubusercontent.com/10895065/44145106-f4eae90a-a092-11e8-87ec-0d8294ab87c7.png)

but definitely not for a table